docs/developer/app-store/apps/taxes.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Taxes
+sidebar_position: 2
+---
+
+# Taxes
+
+## Overview
+
+_Taxes_ is a Saleor app that allows delegating tax calculations to external tax providers. Here are the currently available providers:
+
+- [TaxJar](https://www.taxjar.com/)
+- [Avatax](https://www.avalara.com/us/en/products/calculations.html)
+- [Avatax Excise](https://developer.avalara.com/excise/) [_coming soon_]
+
+Taxes App has two views: **Channels** and **Providers**.
+
+**Channels** allows you to fill in the "ship from" address and assign a tax provider instance to the channel.
+
+**Providers** allows you to manage the tax provider instances. You can have multiple instances of the same provider.
+
+:::caution
+
+To successfully configure the Taxes App, you must have an account with one of the supported tax providers.
+
+:::
+
+## Development
+
+To run the Taxes App locally:
+
+1. Follow the [_Setup_ section in the _Development_ article](developer/app-store/development.mdx#setup).
+2. Go to the app directory.
+3. Copy the `.env.example` file to `.env`.The `.env` should contain the following variables:
+
+`SECRET_KEY` (**required**)
+
+A randomly generated key for the encryption of [Settings Manager](https://github.com/saleor/saleor-app-sdk/blob/main/docs/settings-manager.md).
+
+`APL` (_optional_)
+
+Name of the chosen implementation of the [Authentication Persistence Layer](https://github.com/saleor/saleor-app-sdk/blob/main/docs/apl.md).
+
+When no value is provided, `FileAPL` is used by default. See `saleor-app.ts` in the app directory to see supported APLs.
+
+`APP_DEBUG` (_optional_)
+
+Logging level based on which the app will decide on what messages to log.
+
+The possible values are: `fatal`, `error`, `warn`, `info`, `debug`, `trace`. Read more in [pino documentation](https://getpino.io/#/docs/api?id=loggerlevel-string-gettersetter).
+
+`NODE_ENV` (_optional_)
+
+Controls the environment in which the app is running.
+
+The possible values are: `development`, `production`.
+
+`PORT` (_optional_)
+
+The port on which the app will run. `3000` by default.
```

Comment:
I'd not mentions them, but mention that app is using next js and link to docs of next for others

https://nextjs.org/docs/basic-features/environment-variables